src/material/list/list-item.html

```diff
@@ -1,4 +1,4 @@
-<div class="mat-list-item-content">
+<span class="mat-list-item-content">
```

Comment:
You need to update the CSS to make the spans `display: block` or else this will cause visual changes

Reply:
This span already has `display: flex`, from src/material/list/list.scss